content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -2,59 +2,64 @@
 Help Center
 ===========
 
-Odoo *Helpdesk* integrates with the *Forum*, *eLearning*, and *Knowledge* apps to create the *Help
-Center*. The *Help Center* is a centralized location where teams and customers can search for and
-share detailed information about products and services.
+.. |plus| replace:: :icon:`fa-plus` :guilabel:`(plus)` icon
+
+Odoo **Helpdesk** integrates with the **Forums**, **eLearning**, and **Knowledge** apps to create
+the *Help Center*. The *Help Center* is a centralized location where teams and customers can search
+for and share detailed information about products and services.
 
 .. image:: help_center/help-center-enable-features.png
-   :align: center
    :alt: Overview of the settings page of a team emphasizing the Help Center features.
 
 Configuration
 =============
 
-To activate any *Help Center* features (*Forums*, *eLearning*, or *Knowledge*) on a *Helpdesk* team,
-go to :menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams` and select a team, or
-create a :doc:`new one <../../helpdesk>`. Verify the :guilabel:`Visibility` of the team is set to
+To activate any of the features *Help Center* features on a *Helpdesk* team, go to
+:menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams` and select a team, or create a
+:doc:`new one <../../helpdesk>`. Verify the :guilabel:`Visibility` of the team is set to
 :guilabel:`Invited portal users and all internal users (public)` in the :guilabel:`Visibility &
 Assignment` section.
 
-Additionally, the :guilabel:`Website Form` option on the *Helpdesk* team page **must** be enabled to
-activate any of the *Help Center* features. When one or more of the *Help Center* features is
+Additionally, the :guilabel:`Website Form` option on the **Helpdesk** team page **must** be enabled
```

Comment:
Suggestion to state this as the Helpdesk team's record "_form_" rather than "page"
```suggestion
Additionally, the :guilabel:`Website Form` option on the **Helpdesk** team form **must** be enabled
```